spec/problem_two_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../problem_two'
+
+describe ProblemTwo do
+  let(:p_two) { $p_two = ProblemTwo.new }
```

Comment:
Why do you need to do `$p_two = ...` ?

Reply:
It was requested by @drqCode in the previous pull request: 
> Could you use `let`s? Check this: https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let

Reply:
Misunderstood ` let(:count) { $count += 1 }` in https://tinyurl.com/y7aaxlay. Fixed in https://github.com/Agilefreaks/Test/pull/42/commits/bd698d0569424006d3abdafe4524b454c4c5f78c